tests/py/dynamo/converters/test_split_aten.py

```diff
@@ -0,0 +1,220 @@
+import torch
+from harness import DispatchTestCase
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+from torch_tensorrt import Input
+from torch_tensorrt.dynamo.conversion import UnsupportedOperatorException
+
+
+# FIXME: check about implicit and explicit batch
+class TestSplitConverterNoDim(DispatchTestCase):
+    @parameterized.expand(
+        [
+            ("split_size_or_sections_no_dim", 2),
+        ]
+    )
+    def test_split(self, _, split_size_or_tensor):
+        class TestModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, input):
+                out = torch.split(input, split_size_or_tensor)
+                return out
+
+        input = [torch.randn(10).reshape(5, 2)]
+        self.run_test(
+            TestModule(),
+            input,
+            expected_ops={torch.ops.aten.split.Tensor},
+            disable_passes=True,
+        )
+
+    @parameterized.expand(
+        [
+            ("split_size_or_sections_list_no_dim_list", [1, 4]),
+        ]
+    )
+    def test_split_list(self, _, split_size_or_tensor):
+        class TestModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, input):
+                out = torch.split(input, split_size_or_tensor)
+                return out
+
+        input = [torch.randn(10).reshape(5, 2)]
+        self.run_test(
+            TestModule(),
+            input,
+            expected_ops={torch.ops.aten.split_with_sizes.default},
+            disable_passes=True,
+        )
+
+    @parameterized.expand(
+        [
+            ("split_size_or_sections_list_no_dim_not_full_list", [1, 3]),
+        ]
+    )
+    def test_split_not_full_list(self, _, split_size_or_tensor):
+        class TestModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, input):
+                out = torch.split(input, split_size_or_tensor)
+                return out
+
+        input = [torch.randn(10).reshape(5, 2)]
+        with self.assertRaises(RuntimeError):
+            self.run_test(
+                TestModule(),
+                input,
+                expected_ops={torch.ops.aten.split_with_sizes.default},
+                disable_passes=True,
+            )
```

Comment:
Could these all be the same test case, with something like:
```python
    @parameterized.expand(
        [
            ("split_size_or_sections_list_no_dim_not_full_list", [1, 3]),
            ("split_size_or_sections_list_no_dim_list", [1, 4]),
            ("split_size_or_sections_no_dim", 2),
        ]
```

Reply:
I combined the first two. I do not think I can combine the last since the expected_ops in the first two cases is `torch.ops.aten.split_with_sizes.default `and the third is `torch.ops.aten.split.Tensor`